src/urls.py

```diff
@@ -50,13 +53,15 @@ def get_context_data(self, **kwargs):
                     'name': producer.name,
                     'url': producer.url
                 })
-
         context['producer_data'] = json.dumps(context['producer_data'])
+        # Call our default sort helper function
+        context['default_search_results'] = json.dumps(SearchView._default_search())
```

Comment:
one thing to check here, what if something contains a script? when this is used on the frontend is it possible to XSS?

Reply:
Are you saying like in the competition HTML? I do not think that is possible... Would be interesting to look into though